storage/innobase/handler/ha_innodb.cc

```diff
@@ -8558,16 +8561,20 @@ ha_innobase::update_row(
 #ifdef WITH_WSREP
 	if (error == DB_SUCCESS && trx->is_wsrep()
 	    && wsrep_thd_is_local(m_user_thd)
-	    && !wsrep_thd_ignore_table(m_user_thd)) {
-		DBUG_PRINT("wsrep", ("update row key"));
+	    && !wsrep_thd_ignore_table(m_user_thd)
+	    && (thd_sql_command(m_user_thd) != SQLCOM_CREATE_TABLE)
```

Comment:
It's not clear why CREATE TABLE SELECT or LOAD command would update a row. Is this change really necessary, or am I missing something?